### PR TITLE
add user invite e2e test

### PR DESCRIPTION
**Pull request by nathanosdev**
Adds a test for creating and consuming a user invite. Depends on https://github.com/TaggrNetwork/taggr/pull/138

### CODE REVIEW

**Review by taggrx — APPROVED** [review]
Thanks!

File: src/frontend/src/post.jsx [hunk 1]
```diff
@@ -339,3 +339,19 @@ export const Reactions = ({reactionsMap, react}) => {
 };
 
 const skipClicks = elem => elem && (elem.dataset["meta"] == "skipClicks" || skipClicks(elem.parentElement));
+
+const reactionToTestId = id => {
```
Comment (taggrx) [inline src/frontend/src/post.jsx:343]: You said you'll move it to another place in a follow up PR?

**Review by taggrx — DISMISSED** [review]

File: src/frontend/src/post.jsx (hunk 1 above)
Comment (nathanosdev) [reply src/frontend/src/post.jsx:343]: I did.... this must have gotten back in while merging the branch with `master`. Thanks, I'll remove it.

Comment (nathanosdev) [reply src/frontend/src/post.jsx]: Removed now :)

**Review by taggrx — APPROVED** [review]

**Pull request merged**